From 8659e25434e42f89aa23cbf06f83e3531ff4abe4 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Thu, 22 Jul 2021 08:48:02 +0200
Subject: [PATCH] resolver-conf: one small tweak for Python3.9

Fixes:
/etc/resolver/dhcp_host_domain_ng.py:50: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if output is "syslog":
---
 net/resolver-conf/Makefile                     | 2 +-
 net/resolver-conf/files/dhcp_host_domain_ng.py | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/resolver-conf/Makefile b/net/resolver-conf/Makefile
index 908d6a7e6..c5a468c26 100644
--- a/net/resolver-conf/Makefile
+++ b/net/resolver-conf/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=resolver-conf
 PKG_VERSION:=0.0.4
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>
 PKG_LICENSE:=GPL-3.0-or-later
diff --git a/net/resolver-conf/files/dhcp_host_domain_ng.py b/net/resolver-conf/files/dhcp_host_domain_ng.py
index 0c9bcd40d..f782abcee 100755
--- a/net/resolver-conf/files/dhcp_host_domain_ng.py
+++ b/net/resolver-conf/files/dhcp_host_domain_ng.py
@@ -47,7 +47,7 @@ def is_valid_ipv6(address):
 
 
 def log(text, priority=LOG_INFO, output="syslog"):
-    if output is "syslog":
+    if output == "syslog":
         syslog.syslog(priority, text)
     else:
         print(text)
-- 
2.30.2

